qiita_pet/handlers/api_proxy/tests/test_prep_template.py

```diff
@@ -0,0 +1,26 @@
+from unittest import TestCase, main
+
+from qiita_pet.handlers.api_proxy.prep_template import prep_template_get_req
+
+
+class TestPrepAPI(TestCase):
+    def test_prep_template_get_req(self):
+        obs = prep_template_get_req(1, 'test@foo.bar')
+        exp = {'18S': [
+            {'id': 1,
+             'name': 'PREP 1 NAME',
+             'status': 'private',
+             'start_artifact_id': 1,
+             'start_artifact': 'FASTQ',
+             'last_artifact': 'TODO new gui'
+             }]}
+        self.assertEqual(obs, exp)
+
+    def test_prep_template_get_req_no_access(self):
+        obs = prep_template_get_req(1, 'demo@microbio.me')
+        exp = {'status': 'error',
```

Comment:
If you are using the status + message strategy that has been used in other points, the previous test case reflects that the returns are inconsistent. The return in the previous case (success) should be:
`{'status': 'success', 'message': None, 'info': {<previous_result>}}`